src/transform/schema.ts

```diff
@@ -15,10 +15,14 @@ interface TransformSchemaObjOptions extends GlobalContext {
   required: Set<string>;
 }
 
+function hasDefaultValue(node: any): boolean {
+  if (node.hasOwnProperty("default")) return true;
```

Comment:
This is the only change here: check if a schema object has a `default` value. But like the description states, we can’t currently look this up for `$ref` (which might be a remote schema)